llvm/lib/Transforms/Vectorize/VPlanRecipes.cpp

```diff
@@ -966,6 +966,7 @@ bool VPInstruction::onlyFirstLaneUsed(const VPValue *Op) const {
   case VPInstruction::BranchOnCount:
   case VPInstruction::BranchOnCond:
   case VPInstruction::ResumePhi:
+  case VPInstruction::Broadcast:
```

Comment:
This was needed since we're now using VPInstruction::Broadcasts for arbitrary VPValues